<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1069</width>
    <height>721</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListWidget" name="GamemodeList">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>571</width>
     <height>691</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="SettingsButton">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>20</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </widget>
  <widget class="QListWidget" name="MessagingList">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>211</width>
     <height>691</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="FriendsList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>141</width>
     <height>641</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ManageFriendsButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>670</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Manage Friends</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
